playground/config/__tests__/resolveConfig.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { createLogger, normalizePath, resolveConfig } from 'vite'
+import { expect, it } from 'vitest'
+
+const normalizeString = (str: string) => {
+  const myRoot = normalizePath(process.cwd())
+  // eslint-disable-next-line no-control-regex
+  str = str.replace(/\x1b\[[\d;]+m/gi, '').replace(myRoot, '')
+  return str
+}
+
+it('resolveConfig with root path including both "#" and "?" should warn for both charchters ', async () => {
```

Comment:
I think each tests here needs a `expect.assertions(1)` so that we make sure a single assertion is called.